inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/tenant/InlongTenantServiceImpl.java

```diff
@@ -156,6 +167,32 @@ public Boolean delete(String name) {
         String operator = LoginUserUtils.getLoginUser().getName();
         log.info("begin to delete inlong tenant name={} by user={}", name, operator);
         InlongTenantEntity inlongTenantEntity = inlongTenantEntityMapper.selectByName(name);
+        // before deleting a tenant, check if all Groups of the tenant are in stop status
+        List<InlongGroupEntity> groupList = groupMapper.selectByTenant(name);
+        long notStopGroupCount =
+                groupList.stream().filter(
+                        group -> !GroupStatus.DELETED.getCode().equals(group.getStatus()) || group.getIsDeleted() == 0)
+                        .count();
+        if (notStopGroupCount > 0) {
+            String errMsg = String.format(
```

Comment:
Please log out all groups that are not in stop or deleted status

Reply:
finish